This change adds blog content about what Porter is to the README file (#3)
getporter/kubecon-2023: README.md

<img align="right" src="https://github.com/getporter/porter/blob/main/docs/static/images/porter-docs-header.svg" width="300px" />

## What is Porter?

> Porter is the "glue" for all your existing tools to make packaging, distribution and deployment of your software much easier. With that said, porter is not replacing your existing tools, but rather leveraging them and doing some extra nice things on top of them.

Reviewer: ```suggestion
Porter is the "glue" for all your existing tools to make packaging, distribution and deployment of your software much easier. Porter is not replacing your existing tools, but rather leveraging them and doing some extra nice things on top of them.
```

## How to get started
We have prepared a small demo to give you a peek into how Porter can enhance your distribution and deployment experience. You can find it here https://getporter.org/docs/quickstart/bundles/

If your deployments happen on Kubernetes clusters, you can still use Porter while never leaving the Kubernetes way of doing things. To achieve this, Porter is integrated into an operator that executes Porter commands on your behalf when you apply Kubernetes resources onto your cluster. We have prepared a little demo for this as well, https://getporter.org/docs/operator/quickstart/

## Users
Porter is already getting some interesting use cases. We’re going to list the ones we know about below. We also encourage users of Porter out there to reach out to us and tell us about how they’re using Porter.

<img align="left" src="/imgs/microsoft.png" height="108px" width="300px" />

> Porter is used by the [Azure TRE](https://microsoft.github.io/AzureTRE/v0.11.0/azure-tre-overview/architecture/) team where Porter bundles are used to create trusted research environments in hospitals and schools across countries.

Reviewer: ```suggestion
 The [Azure TRE](https://microsoft.github.io/AzureTRE/v0.11.0/azure-tre-overview/architecture/) uses Porter Bundles to create Trusted Research Environments. These bundles are used to enable research in medical facilities and schools from the US to all over Europe!
```

<img align="left" src="/imgs/f5-logo.svg" height=51px" width="50px" />

> At [F5](https://www.f5.com/) Placeholder text... By leveraging Porter, F5 Networks can streamline their deployment processes, ensuring seamless, reliable, and scalable application delivery while adhering to cloud-native best practices. This adoption demonstrates F5 Networks' dedication to staying at the forefront of technology and optimizing their workflows for enhanced efficiency and agility in the rapidly evolving digital landscape.

Reviewer: ```suggestion
At [F5](https://www.f5.com/) By leveraging Porter, F5 Networks can streamline their deployment processes, ensuring seamless, reliable, and scalable application delivery while adhering to cloud-native best practices. This adoption demonstrates F5 Networks' dedication to staying at the forefront of technology and optimizing their workflows for enhanced efficiency and agility in the rapidly evolving digital landscape.
```

Author: This one is still placeholder text. Waiting on Brian to share the f5 white paper

Reviewer: I think this is it https://github.com/bdegeeter/porter-papers/blob/main/share-a-cloud-application.md

Author: That one seems general to Porter. I thought we wanted something specific to F5
